src/auth/auth-models.js

```diff
@@ -0,0 +1,67 @@
+const mongoose = require('mongoose');
+const bcrypt = require('bcrypt');
+
+/** set up auth user model **/
+const authSchema = new mongoose.Schema({
+  email: {
+    type: String,
+    unique: true,
+    required: true
+  },
+  password: {
+    type: String,
+    required: true
+  },
```

Comment:
It's not entirely clear which ```password``` variables are hashed, and which ones are not.  I think the hashed passwords should be called something like ```passwordHash```.